packages/cli/package.json

```diff
@@ -153,6 +155,7 @@
     "@types/tar": "^6.1.1",
     "@types/terser-webpack-plugin": "^5.0.4",
     "@types/yarnpkg__lockfile": "^1.1.4",
+    "cross-fetch": "^3.1.5",
```

Comment:
This doesn't seem to be used? You'll want to stick to `node-fetch` either way in CLI packages